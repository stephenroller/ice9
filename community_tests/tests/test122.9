# type foo = int[2] var a : foo a[0] := 0;				succeed

type foo = int[2]
var a : foo

a[0] := 7;

write a[0];