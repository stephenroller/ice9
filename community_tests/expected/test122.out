Loading file: generated/test122.tm
7 
Number of instructions executed = 48
